### PR TITLE
add simple improved Sieve Of Eratosthenes Algorithm

### PR DESCRIPTION
**Pull request by ShuLaPy**

### CODE REVIEW

**Comment by cclauss** [comment]
https://travis-ci.com/TheAlgorithms/Python/builds/132832134#L366

File: maths/SieveOfEratosthenesPrimeNumber.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+Input : n =10
+Output : 2 3 5 7 
+
+Input : n = 20 
+Output: 2 3 5 7 11 13 17 19
+'''
+
+def primeNumbers(num):
+
```
Comment (cclauss) [inline maths/SieveOfEratosthenesPrimeNumber.py]: Add doctests as discussed in CONTRIBUTING.md:
```python
def prime_numbers(num: int) -> None:
    """
    >>> prime_numbers(10)
    2 3 5 7 
    >>> prime_numbers(20)
    2 3 5 7 11 13 17 19 
    """
```

Comment (ShuLaPy) [reply maths/SieveOfEratosthenesPrimeNumber.py]: added that

**Comment by cclauss** [comment]
https://travis-ci.com/TheAlgorithms/Python/builds/132872849#L365
1 files contain uppercase characters:
maths/SieveOfEratosthenesPrimeNumber.py

**Comment by cclauss** [comment]
This check is on the filename, not function name (although that is good too).

**Comment by ShuLaPy** [comment]
Thank you very much this is my first successful contribution.

**Pull request merged**